joomla/joomla-cms
# [5.0] Batch Copy & Move tags

Pull Request for Issue #23304 .

### Summary of Changes
Solves a four year old feature request ;)

Adds the ability to copy and move tags


https://user-images.githubusercontent.com/1296369/186410752-c120430b-35cf-40fe-ba92-0a713b3d827a.mp4


Its a new feature so its against the 4.3 branch

### Testing Instructions
It requires the tag alias bug fix from #38576
It contains an updated javascript so run npm build:js

See video
